logic/common/utils_file.py

```diff
@@ -0,0 +1,191 @@
+'''
+This file contains:
+ - Custom printing funcitons
+ - Reader function for choose the root from TOML
+
+'''
+
+#==================================================#
+'''Import & Default Constants'''
+
+import time
+import os
+import toml
+
+DEFAULT_ROOT_TOML = "input/root_dir.toml"
+
+
+
+#==================================================#
+'''Print Functions'''
+
+def print_linebreak(upper_spacing = 0, lower_spacing = 0, linebreak_char = '='):
+	'''Prints a divider, can add extra spacing above and/or below'''
+	for spacing_count in range(upper_spacing):
+		print("")
+	print(linebreak_char*50)
+	for spacing_count in range(lower_spacing):
+		print("")
+
+def _print(print_str):
+	'''Alternate print function'''
+	print(f"\t{print_str}")
+
+def _print_list(list_s):
```

Comment:
Hmmm... I'm debating the usefulness of these print functions. Python has a lot of ways to print lists elegantly.

For example, you could use:
`print(' '.join(list_s))`

Or if you want newlines:
`print('\n'.join(list_s))`

Reading material: https://www.geeksforgeeks.org/print-lists-in-python-4-different-ways/#

This function would crash on numbers. Maybe this function can justify its existence by handling both numbers and strings elegantly?

Reply:
Hm... I'm not seeing any crash with either my print_list() or your join() methods, do you mind specifying?

Reply:
Ah, it seems join handles the conversion of numbers to strings, so it's all good.